lib/Dialect/FIRRTL/Transforms/RemoveResets.cpp

```diff
@@ -0,0 +1,125 @@
+//===- RemoveResets.cpp - Ingest black box sources --------------*- C++ -*-===//
```

Comment:
```suggestion
//===- RemoveResets.cpp - Remove resets of invalid value --------*- C++ -*-===//
```

Reply:
Nice catch. I was clearly copy-pasting this... 😉  Will be fixed in the rebase.